src/index.ts

```diff
@@ -114,9 +116,15 @@ export const getGPUTier = async ({
         (response) => response.json()
       );
 
-      // Remove version tag
-      data.shift();
-
+      // Remove version tag and check version is supported
+      const version = parseInt(
+        (data.shift() as unknown as string).split('.')[0]
+      );
+      if (version < 4) {
```

Comment:
What is the breaking change that is introduced in the benchmark data?

Reply:
https://github.com/pmndrs/detect-gpu/blob/00b4c619b5a822ab0b009c9ca87ad0726be48c67/scripts/update_benchmarks.ts#L135